### PR TITLE
🎨 [Frontend] Connect Anatomical modes to Licensed items

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Connect ``licensed_items`` and their pricing plans to ViP Models
- [x] Connect ``purchases`` and implement ``:purchase``
- [x] Pricing plans and units: refactoring and own Store
- [x] Talk to iframe, as discussed:

rocket -> osparc
```
"type": "openMarket",
"message": {
  "category": "human | human_region | animal | phantom",
},
```
osparc -> rocket
```
"type": "importModel",
"message": {
  "modelId": 5,
},
```

Market

## Related issue/s

- related to https://github.com/ITISFoundation/osparc-issues/issues/1732

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/data/Resources.js [hunk 1]
```diff
@@ -1239,6 +1239,23 @@ qx.Class.define("osparc.data.Resources", {
...
+       */
+      "market": {
+        useCache: false,
+        endpoints: {
+          get: {
+            method: "GET",
+            url: statics.API + "/catalog/licensed-items"
+          },
+          purchase: {
+            method: "GET",
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/data/Resources.js:1255]: This is probably a POST

File: services/static-webserver/client/source/class/osparc/vipMarket/AnatomicalModelDetails.js [hunk 2]
```diff
@@ -167,40 +167,42 @@ qx.Class.define("osparc.vipMarket.AnatomicalModelDetails", {
...
+            pricingUnitsLayout.add(pUnit);
+          });
+        })
+        .catch(err => console.error(err));
+
+      return pricingUnitsLayout;
+    },
+
+    __rentAnatomicalModel: function(anatomicalModelsData, pricingUnit) {
+      console.log(":puschase", anatomicalModelsData["licensedItemId"], pricingUnit["pricingUnitId"]);
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/vipMarket/AnatomicalModelDetails.js:205]: "Puscas" was a real madrid player ;-)

**Review by pcrespov — APPROVED** [review]
very nice :-)

File: services/static-webserver/client/source/class/osparc/vipMarket/AnatomicalModelDetails.js (hunk 2 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/vipMarket/AnatomicalModelDetails.js:205]: In this PR there is only one "real" and it's not madrid ;)

**Review by sanderegg — COMMENTED** [review]
nice!

**Review by matusdrobuliak66 — APPROVED** [review]
:100:

**Review by GitHK — COMMENTED** [review]
👍

**Review by jsaq007 — APPROVED** [review]

**Pull request merged**